modules/gdscript/language_server/gdscript_extend_parser.h

```diff
@@ -50,6 +53,63 @@
 
 typedef HashMap<String, const lsp::DocumentSymbol *> ClassMembers;
 
+/**
+ * Represents a Position as used by GDScript Parser. Used for conversion to and from `lsp::Position`
+ *
+ * Difference to `lsp::Position`:
+ * * line & char/column: 1-based
+ * 		* LSP: both 0-based
+ * * tabs are expanded to columns using tab size (`text_editor/behavior/indent/size`)
+ *   	* LSP: tab is single char
+ *
+ * Example:
+ * ```gdscript
+ * →→var my_value = 42
+ * ```
+ * `_` is at:
+ * * Godot: `column=12`
+ * 	* using `indent/size=4`
+ * 	* Note: counting starts at `1`!
+ * * LSP: `character=8`
+ * 	* Note: counting starts at `0`!
+ */
+struct GodotPosition {
+	int line;
+	int column;
+
+	GodotPosition(int line, int column) :
```

Comment:
```suggestion
	GodotPosition(int p_line, int p_column) :
```
Use `p_` prefix for function parameters, to prevent shadowing